pkg/plugin/processor/builtin/excludeFields_test.go

```diff
@@ -0,0 +1,97 @@
+// Copyright © 2024 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package builtin
+
+import (
+	"context"
+	"testing"
+
+	"github.com/conduitio/conduit-commons/opencdc"
+	sdk "github.com/conduitio/conduit-processor-sdk"
+	"github.com/matryer/is"
+)
+
+func TestExcludeFields_Process(t *testing.T) {
+	is := is.New(t)
+	proc := excludeFields{
+		fields: []string{".Metadata", ".Payload.After.foo"},
```

Comment:
what should we expect when excluding `.Payload.After.something`? the current behaviour would be that nothing changes, since the resolver would reference and then delete it

Reply:
Well, I'd say there are 2 options there:
1. do nothing, because the goal is not to have the field
2. fail, because someone might have expected the field to be there.

I'm leaning towards #1, because #2 is too strict, and it's likely that a user will simply want to normalize some data. If someone _really_ wants to validate the input data, they can write a processor for that.

Reply:
I like 1 too, cuz the user might want to remove a field IF it exists, it might exist in some records and not others